modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -126,12 +126,12 @@ Java equivalents, such as `America/Los_Angeles` and
 
 //source info: +https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html+
 
-== Debugging a Scheduler
+== Stop or Start a Scheduler in Debug Mode
 
-When you debug a Mule application in Anypoint Studio, each Scheduler event source appears with an arrow icon underneath to remind you that the Scheduler can be stopped and started while the Mule application is being debugged. To stop or start the Scheduler event source, right click on the Scheduler and select Stop Scheduler or Start Scheduler from the Scheduler's right-click menu.  
+In Anypoint Studio, when you run a Mule application using debug mode, an arrow icon appears at the bottom of the Scheduler component. The arrow indicates that you can start or stop the Scheduler from its right-click menu.
 
+By default, the Scheduler runs when in debug mode. If you place a break point on a message processor in the flow, the Scheduler component will start the flow and continue emitting events as scheduled until you stop it. If you set a break point before stopping the Scheduler and the Scheduler emits one or more events, the application queues those events to run through the flow after you finish stepping through the break point.
```

Comment:
"starts...continues" not "will start" for simple active verb tense where possible.